src/decoder/mod.rs

```diff
@@ -429,6 +458,36 @@ impl<R: Read + Seek> Decoder<R> {
             0 => None,
             n => Some(n),
         };
+        // A special BigTiff format is generated by ScanImage and is identified
+        // and (partially) parsed under this if statement. It's identified by
+        // matching on a "magic number" that ScanImage defines [here](http://scanimage.vidriotechnologies.com/display/SI2019/ScanImage+BigTiff+Specification#ScanImageBigTiffSpecification-ScanImageStaticMetadataSection).
+        if self.bigtiff {
```

Comment:
In general I would say that this should require an 'opt-in' of some sort. As far as I understand, the ifd offset is non-zero such that generic big tiff reading can simply skip over the extra metadata. But what's to say that some other image does not start with the magic number by chance anyways? Then this might error where it previously did not.

I'd propose adding a field `read_scan_image: bool` that starts as `false` and which you explicitly activate by calling a method similar to limits.

> `pub fn with_scan_image(mut self) -> Decoder<R> `

Then this check would become:
```suggestion
        if self.bigtiff && self.read_scan_image {
```